config/jobs/kubernetes/sig-node/dynamic-resource-allocation-canary.yaml

```diff
@@ -0,0 +1,558 @@
+# This file contains canary periodic and presubmit jobs which run tests covering
+# Dynamic Resource Allocation (DRA).
+#
+# The intent is to make all changes to DRA jobs first for these canary jobs,
+# then copy the changes into dynamic-resource-allocation.yaml. Unless some
+# experimental changes are being tested, a diff between the two files should
+# be limited to:
+# - this comment
+# - job names
+# - interval for the periodic jobs
+#
+# This command can be used to check this:
+#    diff dynamic-resource-allocation.yaml <(sed -e 's/-canary//' dynamic-resource-allocation-canary.yaml)
+
+periodics:
+
+  # This jobs runs e2e.test with a focus on tests for the Dynamic Resource Allocation feature (currently beta)
+  # on a kind cluster with containerd updated to a version with CDI support.
+  - name: ci-kind-dra-canary
+    cluster: eks-prow-build-cluster
+    interval: 1000000h # Run only once on creation and when manually triggered.
```

Comment:
/hold

I am not sure yet how this "manually triggered" could work. I asked on Slack: https://kubernetes.slack.com/archives/C09QZ4DQB/p1734506374693689